lambda/server.js

```diff
@@ -0,0 +1,32 @@
+const env = require('dotenv');
+const express = require('express');
+const bodyParser = require('body-parser');
+const routeToPhoenix = require('./routeToPhoenix');
+const app = express();
+
+env.config();
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+const PORT = 4400;
+
+app.post('/', async (req, res) => {
+  const testUrl = 'http://localhost:4000/graphql'; // phoenix development server
```

Comment:
How did you test this?

Reply:
spin up a local test server, and hit it with Postman
tried my best to simulate how the lambda function would be invoked with payload `Content-Type` = `application/json`, and identical payload data signature